README.md

```diff
@@ -46,7 +46,9 @@ In February 2020 Cedarville Unversity agreed to move it to GitHub with @joe-sonr
 
 2. Next, go to the root of the git repo.
 
-3. Run `mkdir build && cd build && cmake .. "Visual Studio 15 2017"`.
+3. Run `mkdir build && cd build`
+
+3. Run `cmake .. -A Win32` (If you did not set the windows environment variable earlier, also add `-Dwxwin=C:/wxWidgets-2.8.12` to set `wxwin` variable to the base of the wxWidgets install path).
 
 4. There is now a visual studio solution in the `build` directory. Open the solution file.
```

Comment:
Must be 4. instead of 3.

Reply:
This is a markdown file. The file renderer will redo the numbering itself no matter what you put better. You can (and probably should) just put `1.` for each item in the list.

See the file as it renders and observe that it "did the right thing":
https://github.com/CedarvilleCS/CedarLogic/blob/a8861bd1f26fb13891ef4e21b105ff08aee0b999/README.md

I left these duplicated on purpose because this is a pretty long list and would generate a lot of useless changed lines if we were to fix them all.

Reply:
Yeah, according to https://github.com/markdownlint/markdownlint the technically correct way is to make everything a `1.` but a lot of renderers (such as those used by GitHub and GitLab) are pretty forgiving.